03/app.js

```diff
@@ -7,12 +7,51 @@ const c = randomNumber(min, max);
 
 console.log(a, b, c);
 
-
-
-
-
-
-
 function randomNumber(min, max) {
     return Math.round((Math.random() * (max - min)) + min);
-}
\ No newline at end of file
+}
+
+function getSum(d, e, f) {
+    const dInt = parseInt(d);
+    const eInt = parseInt(e);
+    const fInt = parseInt(f);
+
+    const arr = [dInt, eInt, fInt];
+    arr.sort();
```

Comment:
Warto podawać sposób sortowania, obecnie jest to sortowanie jak dla ciągów znaków (napisów) i prawdopodobnie sortowania dla liczb: `221, 1, 3, 23, 33, 11` nie będzie prawidłowe.